packages/hydra-cli/src/commands/scaffold.ts

```diff
@@ -1,31 +1,73 @@
 import { Command, flags } from '@oclif/command'
 import * as fs from 'fs-extra'
 import * as path from 'path'
-import * as utils from './../utils/utils'
 import cli from 'cli-ux'
-import { getTemplatePath } from '../utils/utils'
-import Mustache = require('mustache')
-import dotenv = require('dotenv')
-import execa = require('execa')
+import Mustache from 'mustache'
+import Debug from 'debug'
+
+import {
+  getTemplatePath,
+  createDir,
+  resolvePackageVersion,
+  getWarthogDependency,
+} from '../utils/utils'
+
+import select = require('@inquirer/select')
+import input = require('@inquirer/input')
+import password = require('@inquirer/password')
 import glob = require('glob')
 
-const DEFAULT_WS_API_ENDPOINT = 'wss://kusama-rpc.polkadot.io/'
-const KUSAMA_INDEXER = 'https://indexer-kusama.joystream.app/graphql'
+const debug = Debug('qnode-cli:scaffold')
+
+// TODO: fetch from a well-known source?
+const INDEXERS = [
+  {
+    name: 'local',
+    value: 'localhost',
+    description: 'Self-hosted indexer at localhost:4000',
```

Comment:
Perhaps not so important but the `description` shows a different port than `url`